src/continuum_absorption/absorption_ff_metal.jl

```diff
@@ -0,0 +1,458 @@
+departure term is specified.
+
+A free-free interaction is named as though the species interacting with the free electron had one 
+more bound electron (in other words it's named as though the free-electron and ion were bound 
+together). The `Z` argument and `ni` values should be specified for the species that actually
+participates in the reaction. For completeness 2 examples provided below:
+- Si I ff absorption: `ni` holds the number density of Si II, and `Z=1` (net charge of Si II)
+- Si II ff absorption: `ni` holds the number density of Si III, and `Z=2` (net charge of Si III)
+
+# Arguments
+- `ν`: frequency in Hz
+- `T`: temperature in K
+- `Z::Integer`: the net charge of the ion species (that participates in the interaction).
+- `ni`: the number density of the ion species (that participates in the interaction) in cm⁻³.
+- `ne`: the number density of free electrons.
+- `departure`: a function or functor that gives the departure (i.e. relative difference from
+  hydrogenic absorption) value at a given Temperature and σ, where 
+  σ = (ν / Hz) / (Z² Rydberg_eV / hplanck_eV).
+
+# Notes
+
+This calculation is based on the approach described in
+[Peach 1970](https://ui.adsabs.harvard.edu/abs/1970MmRAS..73....1P/abstract).
+In short, the free-free absorption (including stimulated emission) is given by:
+
+``\\alpha_{\rm ff} = \\alpha_{\rm hydrogenic, ff}(\\nu, T, n_i, n_e; Z) (1 + D(T, \\sigma))``,
+
+where
+- ``\\alpha_{\rm hydrogenic, ff}(\\nu, T, n_i, n_e; Z)`` should includes the correction for 
+  stimulated emission.
+- ``D(T, \\sigma)`` is specified as the `departure` arg, and is expected to interpolate over
+the tabulated values specified in Table III of Peach (1970).
+
+It might not be immediately obvious the above equation relates to the equations presented in
+Peach (1970). Peach describes the calculation for ``k_\nu^F``, the free-free absorption 
+coefficient (uncorrected for stimulated emission) per particle of the species that the interaction 
+is named after. In other words, he computes:
+ 
+``k_\nu^F = \\alpha_{\rm ff}/n_{i-1} \\left(1 - e^\\frac{-h\\nu}{k T}\\right)^{-1}``,
+
+where ``n_{i-1}`` is the number density of the species that the interaction is named after.
+``k_\nu^F`` can directly be computed, under LTE, from just ``\nu``, ``T``, and ``n_{i-1}`` (the
+Saha Equation relates ``\\alpha_{\rm ff}``'s dependence on ``n_e`` and ``n_i`` to ``n_{i-1}`` and 
+``T``.  Gray (2005) follows a similar convention when describing free-free absorption.
+
+As an aside, a ``D(T, \\sigma)`` that is zero everywhere, is equivalent to just using a hydrogenic
+approximation.
+"""
+function metal_ff_absorption_departure(ν::Real, T::Real, Z::Integer, ni::Real, ne::Real, departure)
+    σ = ν/ Z^2 * (hplanck_eV / Rydberg_eV)
+    hydrogenic_ff_absorption(ν, T, Z, ni, ne) * (1 + departure(T, σ))
+end
+
+
+# should we be loading this from a hdf5 table?
```

Comment:
imho it's preferable to do this for the sake of package load time when tables are small.  I do think it would be a good idea to put these in a separate ".jl" though.